deepchecks/vision/utils/transformations.py

```diff
@@ -118,24 +147,59 @@ def get_robustness_augmentations(cls, data_dim: Optional[int] = 3):
         return augmentations
 
 
-def get_transforms_handler(transforms):
+class TorchTransformationsClassification(AlbumentationsTransformations):
+    """Class containing supporting functions for torch transforms."""
+
+    @classmethod
+    def add_augmentation_in_start(cls, aug, transforms: T.Compose):
+        """Add given transformations to the start of given transforms object."""
+        if isinstance(aug, (albumentations.Compose, albumentations.BasicTransform)):
+            alb_aug = aug
+
+            class TorchWrapper:
+                def __call__(self, image):
+                    if isinstance(image, Image.Image):
+                        return T.ToPILImage()(alb_aug(image=np.array(image))['image'])
+                    elif isinstance(image, torch.Tensor):
+                        image = image.cpu().detach().numpy()
+                        return T.ToTensor()(alb_aug(image=image)['image'])
+                    return alb_aug(image=image)['image']
+            aug = TorchWrapper()
+        elif not isinstance(aug, torch.nn.Module):
+            raise DeepchecksValueError(f'Transforms is of type torch, can\'t add to it type '
+                                       f'{type(aug).__name__}')
+        return T.Compose([aug, *transforms.transforms])
+
+
+class TorchTransformations(TorchTransformationsClassification):
```

Comment:
1. Instead of having a different class maybe add task_type to `get_robustness_augmentations`
2. If you really want 2 classes, makes more sense that `TorchTransformationsClassification` will inherit from `TorchTransformations` and not the opposite

Reply:
I want it to inherit from classification because then I can filter the transforms

Reply:
so inherit both from Albumentations. this is too confusing IMO it's like `VisionData` will inherit from `DetectionData`

Reply:
but both using same add_aug_at_start function

Reply:
and I don't want to add task type as parameter, it make the usage more wonky and I need it in the class